bin/www

```diff
@@ -13,5 +13,5 @@ app.set 'port', process.env.PORT
 
 console.log "Environment: #{app.get('env')}"
 
-server = app.listen app.get('port'), ->
+server = app.listen require('systemd-socket')() || app.get('port'), ->
```

Comment:
The dependency originally was not optional, but now made so with a try/catch.

The `systemd-socket` package automatically detects whether the application was run from systemd and acts accordingly. When it does not respond to socket activation, it uses the port exactly like it did without the change.